integration-tests/pom.xml

```diff
@@ -26,6 +26,14 @@
             <artifactId>quarkus-tika</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.quarkiverse.poi</groupId>
+            <artifactId>quarkus-poi</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.fusesource.jansi</groupId>
+            <artifactId>jansi</artifactId>
+        </dependency>
```

Comment:
Which parser requires this dependency?

Reply:
I removed it in [`4ccc9ca` (#84)](https://github.com/quarkiverse/quarkus-tika/pull/84/commits/4ccc9cab0e2532de7bcb91df6032e035a1bdc137)